src/targets/maven.ts

```diff
@@ -401,14 +415,18 @@ export class MavenTarget extends BaseTarget {
   private getTargetFilename(distDir: string): string {
     const moduleName = parse(distDir).base;
 
-    const isAndroidDistDir = this.mavenConfig.android.distDirRegex.test(
-      moduleName
-    );
-    return isAndroidDistDir
-      ? moduleName.replace(
+    if (this.mavenConfig.android) {
+      const isAndroidDistDir = this.mavenConfig.android.distDirRegex.test(
+        moduleName
+      );
+      if (isAndroidDistDir) {
+        return moduleName.replace(
           this.mavenConfig.android.fileReplaceeRegex,
           this.mavenConfig.android.fileReplacerStr
-        )
-      : `${moduleName}.jar`;
+        );
+      }
+    }
```

Comment:
You should probably print a warning here according to your updated docs?

Reply:
There's a warning when checking for the android config ([line 96](https://github.com/getsentry/craft/blob/e4dc758b79cb9ce6b05ecfca9a80037036f4f3c9/src/targets/maven.ts#L96-L98)), I find it better than doing it for every distribution. Do you think there should be another warning here?

Reply:
Nah, that's alright. Thanks for the info! 🙂